helmfile.d/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -162,6 +162,9 @@ alertmanager:
     resources: {{- toYaml .Values.prometheus.alertmanagerSpec.resources | nindent 6 }}
     storage: {{- toYaml .Values.prometheus.alertmanagerSpec.storage | nindent 6 }}
     topologySpreadConstraints: {{- toYaml .Values.prometheus.alertmanagerSpec.topologySpreadConstraints | nindent 4 }}
+{{- if .Values | get "prometheus.alertmanagerSpec.image" nil }}
+    image: {{- .Values | get "prometheus.alertmanagerSpec.image" | toYaml | nindent 6 }}
```

Comment:
Is there any reason to do a `get` here or could you just do `{{- toYaml .Values.prometheus.alertmanagerSpec.image | nindent 6 }}`?

Reply:
Similarity to the `if` line, but otherwise probably not. Shall I change it?

Reply:
I think it makes sense to change it, since that is what we do in the other lines above.

Reply:
Done.

But the `if` condition makes sense in this form?

Reply:
I think so. There are other ways of doing it, but this looks fine to me.